platforms/quorum/test/awsstorageclass/aws-storageclass.yaml

```diff
@@ -0,0 +1,9 @@
+kind: StorageClass
```

Comment:
the 'test' folder should have been cleaned up and not checked-in

Reply:
@sownak have now removed the 'test' folder through the Cleanup.yml script